crates/formality-prove/src/test/occurs_check.rs

```diff
@@ -20,13 +20,13 @@ fn decls() -> Decls {
 fn direct_cycle() {
     test_prove(decls(), term("exists<ty A> {} => {A = Vec<A>}")).assert_err(
             expect![[r#"
-                judgment `prove_wc_list { goal: {?ty_0 = Vec<?ty_0>}, assumptions: {}, env: Env { variables: [?ty_0], bias: Soundness }, decls: decls(222, [trait Foo <ty> ], [impl <ty> Foo(Vec<^ty0_0>)], [], [], [], [], {}, {}) }` failed at the following rule(s):
```

Comment:
we should probably cleanup the `Env` Debug impl to e.g. suppress empty lists...

Reply:
#187